tests/unit/core/middleware/test_cspMiddleware.js

```diff
@@ -5,15 +5,116 @@ import parse from 'content-security-policy-parser';
 import { csp } from 'core/middleware';
 import { getFakeConfig, getFakeLogger } from 'tests/unit/helpers';
 
+const deployedEnvs = ['dev', 'production', 'stage'];
+
+const cdnHosts = {
+  dev: 'https://addons-dev-cdn.allizom.org',
+  stage: 'https://addons-stage-cdn.allizom.org',
+  production: 'https://addons.cdn.mozilla.net',
+};
+
+const apiHosts = {
+  dev: 'https://addons-dev.allizom.org',
+  stage: 'https://addons.allizom.org',
+  production: 'https://addons.mozilla.org',
+};
+
 describe(__filename, () => {
-  describe('CSP Middleware', () => {
-    const existingNodeEnv = process.env.NODE_ENV;
+  const existingNodeEnv = process.env.NODE_ENV;
+  const existingNodeAppInstance = process.env.NODE_APP_INSTANCE;
+
+  afterEach(() => {
+    process.env.NODE_ENV = existingNodeEnv;
+    delete process.env.NODE_APP_INSTANCE;
+    if (process.env.NODE_APP_INSTANCE) {
```

Comment:
This needs to check the previous value in order to know if it must be reset:

```suggestion
    if (existingNodeAppInstance) {
```